ui/component/commentsList/view.jsx

```diff
@@ -29,12 +29,31 @@ function CommentList(props: Props) {
     fetchComments(uri);
   }, [fetchComments, uri]);
 
+  function sortByParent(arrayOfComments) {
```

Comment:
We should move this logic into the redux selector, so the result of it can be cached, instead of resorting every time.